openshift-kubecost-install.md

```diff
@@ -0,0 +1,513 @@
+Install Kubecost on Redhat Openshift cluster
+==================
+
+## Overview
+
+This documentation provides instructions on how to deploy Kubecost on Redhat Openshift cluster. Additional security settings may also be required. You can refer to the Architecture overview and installation section for more information. 
+
+---
+## Architecture overview:
+
+Currently there are 2 main options to deploy Kubecost on Redhat Openshift Cluster (OCP).
+
+### Standard deployment:
+
+Kubecost is installed with Cost Analyzer and Prometheus as a time-series database. Data is gathered by the Prometheus installed with Kubecost (user-defined Prometheus). Other metrics are scraped by user-defined Prometheus from OCP monitoring stack managed components like KSM, OSM, CAdvisor, etc .... Kubecost then push and query metrics to/from user-defined Prometheus. Enterprise set up could also work with Thanos as additional component.
+
+The standard deployment is illustrated in the following diagram:
+
+![Standard deployment](images/standard-lightbg.png)
+
+### Grafana managed Prometheus deployment:
+
+Kubecost is installed with the core components only (cost model, frontend) without user-defined Prometheus and other components. Grafana agent is installed as part of the solution to scrape the metrics from OCP monitoring stack managed components and Kubecost `/metrics` endpoint to write the data back to the Grafana Cloud managed Prometheus (Grafana Prometheus) instance. Kubecost reads the metrics directly from Grafana managed Prometheus.
+
+The Grafana managed Prometheus deployment is illustrated in the following diagram:
+
+![Grafana managed Prometheus deployment](images/grafana-agent-lightbg.png)
+
+## Installation instructions:
+
+### Standard deployment:
+
+**1. Clone this repository to your dev environment.**
+
+`git clone https://github.com/kubecost/openshift-helm-chart.git`
```

Comment:
Great idea ! To confirm, the one with RAW file works because I do not make that repo as a Helm repo. Btw, we can make it changes in upcoming release which will use the main repo.